serverless.yml

```diff
@@ -0,0 +1,23 @@
+org: Data-Epic
+app: sol-sight-serverless
+service: sol-sight-serverless
+
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: python3.9
+  iamRoleStatements:
+  - Effect: Allow
+    Action:
+      - firehose:DeleteDeliveryStream
+      - firehose:PutRecord
+      - firehose:PutRecordBatch
+      - firehose:UpdateDestination
+    Resource: '*'
+
+functions:
```

Comment:
you would reference the function just like you did for resources

Reply:
Oh...I will put it in place